src/main/kotlin/rys/ajaxpetproject/configuration/MongoConfig.kt

```diff
@@ -0,0 +1,70 @@
+package rys.ajaxpetproject.configuration
+
+import org.springframework.boot.CommandLineRunner
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+import org.springframework.data.mongodb.repository.config.EnableMongoRepositories
+import rys.ajaxpetproject.model.MongoChat
+import rys.ajaxpetproject.model.MongoMessage
+import rys.ajaxpetproject.model.MongoUser
+import rys.ajaxpetproject.repository.ChatRepository
+import rys.ajaxpetproject.service.ChatService
+import rys.ajaxpetproject.service.MessageService
+import rys.ajaxpetproject.service.UserService
+import java.util.*
+
+@Configuration
+@EnableMongoRepositories(basePackageClasses = [ChatRepository::class])
+@ComponentScan(basePackageClasses = [ChatRepository::class])
+class MongoConfig {
+
+    @Bean
+    fun init( userService: UserService,  chatService: ChatService,  messageService: MessageService): CommandLineRunner {
+        return CommandLineRunner {
+            userService.deleteUsers()
+            chatService.deleteChats()
+            messageService.deleteMessages()
+
+            val mongoUsers = listOf(
+                MongoUser(userName = "Alice", password = "pass1"),
+                MongoUser(userName = "Bob", password = "pass2"),
+                MongoUser(userName = "Carol", password = "pass3"),
+                MongoUser(userName = "Dave", password = "pass4"),
+                MongoUser(userName = "Eve", password = "pass5")
+            ).map { userService.createUser(it) }
+            println(mongoUsers)
+
+            val mongoChat1 =
+                chatService.createChat(
```

Comment:
```suggestion
            val mongoChat1 = chatService.createChat(
```
no need for like break here

Reply:
Done